libgloss/arc/hl/hl_api.h

```diff
@@ -0,0 +1,71 @@
+/*********************************************************************
+Copyright  2002-2020 Synopsys Inc.
+*********************************************************************/
+
+#include <stdint.h>
+#include "hl_toolchain.h"
+
+#ifndef _HL_API_H
+#define _HL_API_H
+
+#ifdef __CPLUSPLUS__
+extern "C" {
```

Comment:
we don't use c++ compiler, you can remove this.

Reply:
done